### PR TITLE
Fix little running balance visual bug

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by cjdenio** [comment]
Thanks! Can you share a screenshot to confirm this works?

**Comment by sampoder** [comment]
Screenshot 2023-11-20 at 9 51 14 AM

The funny thing I learnt is that we actually use https://github.com/hackclub/hcb/blob/main/app/views/events/_pending_fee_transaction.html.erb to display fee transactions and not `_canonical_pending_transaction.html`... which is why my original idea was wrong

File: app/views/canonical_pending_transactions/_canonical_pending_transaction.html.erb [hunk 1]
```diff
@@ -147,6 +147,9 @@
   </td>
   <% if @show_running_balance %>
     <td></td>
+    <% if pt.raw_pending_bank_fee_transaction_id %>
+      <td></td>
+    <% end %>
```
Comment (cjdenio) [inline app/views/canonical_pending_transactions/_canonical_pending_transaction.html.erb]:
```suggestion
```
"Actual" fee transactions have a Details button so this isn't needed

Comment (sampoder) [reply app/views/canonical_pending_transactions/_canonical_pending_transaction.html.erb]: good point!

**Review by cjdenio — APPROVED** [review]
looks good to me!

**Pull request merged**